src/pages/error/index.tsx

```
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router-dom'
+
+import { LeftArrow } from '@/assets/icons'
+import { Error, Header, SvgButton } from '@/shared/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)